src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -463,7 +463,7 @@ private void postClimb() {
   public void setMotorCleanUp() {
     m_extender.setMotorCoast();
     // m_basePivot.setMotorCoast();
-    m_gripperPivot.setMotorCoast();
+    // m_gripperPivot.setMotorCoast();
```

Comment:
We want to keep the gripper in brake mode so that it doesn't smash into itself when disabled.